frontend/device/kindle/powerd.lua

```diff
@@ -89,4 +89,10 @@ function KindlePowerD:afterResume()
     end
 end
 
+function KindlePowerD:toggleSuspend(status)
+    if self.lipc_handle then
+        self.lipc_handle:set_int_property("com.lab126.powerd", "powerButton", status)
```

Comment:
lgtm, but @NiLuJe actually knows things about lipc

Reply:
I actually haven't checked what passing 0 to it does. I think it'll do nothing, but I'll have to check.

Setting it to 1 does work when it's "asleep" for a wakeup, though.

TL;DR: This should probably always set it to 1 without accepting an input variable.

Reply:
Yes, I just checked, passing 0 does emulate button too